roles/pulp/templates/settings.py.j2

```diff
@@ -1,3 +1,3 @@
 {% for setting, value in merged_pulp_settings.items() %}
-{{ setting | upper }} = {{ value | to_json }}
+{{ setting | upper }} = {{ value | string | capitalize if value | string in ["True", "False"] else value | to_json }}
```

Comment:
This capitalize does not seem to do anything, if the condition for it to be called is met.
If the problem is, that yamls booleans are incompatible to pythons, we should maybe use a `to_python` filter?
(In doubt, it should be fairly easy to write one.)

Reply:
You are correct that the problem this solves is caused by the use of the wrong filter. We would need to write our own 'to_python' filter and then have ansible load it somehow. We should file an issue for this, but it would not be a priority at this time.